Documentation/git-config.txt

```diff
@@ -235,8 +235,12 @@ FILES
 If not set explicitly with `--file`, there are four files where
 'git config' will search for configuration options:
 
+$PROGRAMDATA/Git/config::
+	System-wide configuration file for *all* Git-related software running
```

Comment:
For writing, there is no order: you have to specify *which* config you want to write to (default is the repository config).